k8s/namespaces/ccd/ccd-case-management-web/perftest-redirect.yaml

```diff
@@ -0,0 +1,21 @@
+# this is to apply permanent redirect on www-ccd.perftest.platform.hmcts.net
+---
+apiVersion: extensions/v1beta1
```

Comment:
You may want to have this not kustomized? Guessing that’s why checks are failing

Reply:
cheers, trying to do this in a meeting didn't return the result :D